src/app/clusters/general-commissioning-server/general-commissioning-server.cpp

```diff
@@ -95,6 +99,28 @@ CHIP_ERROR GeneralCommissioningAttrAccess::Read(const ConcreteReadAttributePath
     case SupportsConcurrentConnection::Id: {
         return ReadSupportsConcurrentConnection(aEncoder);
     }
+#if CHIP_CONFIG_TC_REQUIRED
+    case TCAcceptedVersion::Id: {
+        Optional<TermsAndConditions> outTermsAndConditions;
+        Server::GetInstance().GetTermsAndConditionsProvider()->GetAcceptance(outTermsAndConditions);
+        return !outTermsAndConditions.HasValue() ? aEncoder.Encode(0) : aEncoder.Encode(outTermsAndConditions.Value().version);
```

Comment:
My compiler cannot determine which version of the Put() function to use because the call to Put is ambiguous.
Should there be a casting to the zero value in aEncoder.Encode(0)?

Reply:
Someone else told me the same thing. I just updated the code with a static_cast to uint16_t. I think that should resolve your compilation issue. Thanks for reporting.